ReactViewControl/IViewModule.cs

```diff
@@ -22,7 +23,7 @@ public interface IViewModule {
 
         KeyValuePair<string, object>[] PropertiesValues { get; }
 
-        void Bind(IFrame frame, IChildViewHost host = null);
+        void Bind(IFrame frame, IChildViewHost host = null, Uri devServerUri = null);
```

Comment:
Hum.. This is something internal, we shouldn't have to expose this to the public API

Reply:
you suggested this change some time ago :S

Reply:
any alternative suggestion?